rj_gameplay/stp/rc.py

```diff
@@ -15,7 +15,7 @@ class Robot:
     to enforce that instances of this class should not be mutated."""
 
     __slots__ = [
-        "__id", "__is_ours", "__pose", "__twist", "__ball_sense_triggered",
```

Comment:
I think I commented on this in a previous PR as well, why is this being removed again? How will ball-sense be used in the determination of possession, settling, etc if it's not included in worldstate?